source/manual/2nd-line.html.md

```diff
@@ -60,3 +125,12 @@ Follow these Slack channels while working on Technical 2nd Line:
 [install our Chrome extension]: https://github.com/alphagov/govuk-secondline-blinken#chrome-extension
 [PagerDuty]: https://governmentdigitalservice.pagerduty.com
 [Zendesk]: https://govuk.zendesk.com
+[You can check the Technical 2nd Line rota to find out when your shift is]:https://docs.google.com/spreadsheets/d/1OTVm_k6MDdCFN1EFzrKXWu4iIPI7uR9mssI8AMwn7lU/edit#gid=1297388378
```

Comment:
Still think this should use an [alias link](https://github.com/alphagov/govuk-developer-docs/pull/3668#discussion_r971901229) instead